tox.cover.sh

```diff
@@ -25,6 +25,8 @@ cover () {
     min=98
   elif [ "$1" = "certbot_dns_digitalocean" ]; then
     min=98
+  elif [ "$1" = "certbot_dns_google" ]; then
+    min=98
```

Comment:
I missed this for Digital Ocean but it looks like we could get away with `min=99` here to help ensure future changes to this plugin have test coverage.